.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,53 @@
+name: Bug report
+description: Create a report to help us improve
+title: '[Bug]: '
```

Comment:
Most "bug" claims are non-bug, so probably we shouldn't have this [Bug] header

Reply:
Possibly so.
Also makes sure people don't forget to add a title :)